roles/BCPC-Hadoop-Head-MapReduce.json

```diff
@@ -1,6 +1,13 @@
 {
   "name": "BCPC-Hadoop-Head-MapReduce",
   "default_attributes": {
+    "bcpc": { 
+      "hadoop": {
+        "oozie": {
+          "sharelib_checksum": "updated_automatically"
```

Comment:
I would prefer for `oozie.rb` to handle this attribute being unset (or having a cookbook default attribute) rather than putting a required piece of info in a role. If someone included the Oozie recipe they would also need to set this in some one-off way.